src/stylesheets/components/_top-fold.scss

```diff
@@ -86,6 +90,30 @@ main {
   transition: transform 0.2s ease-in-out;
 }
 
+.yellow-tablet {
+  position: absolute;
+  right: 0;
+  top: 0;
+  transform: scale(0.9) translate(300px, 0) rotate(1500deg);
+  width: 60px;
+}
+
+.green-tablet {
+  position: absolute;
+  right: 0;
+  top: 0;
+  transform: scale(0.7) translate(-100px, -200px) rotate(1500deg);
+  width: 50px;
+}
+
+.purple-tablet {
+  position: absolute;
+  right: 0;
+  top: 0;
+  transform: scale(0.9) translate(-200px, -200px) rotate(1500deg);
```

Comment:
Curious to know why 1500deg is necessary? That's a 4x rotation. Is that for the initial load transition?

Reply:
Yeah that's for the initial load transition - is there a different way to get the same result?

Reply:
No. I forgot that you were spinning the pills on page load.